src/DataProtection/DataProtection/src/Microsoft.AspNetCore.DataProtection.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Accidental BOM change? I'm not sure if it matters though.

Reply:
We usually don't fight editor with these. Letting this be.